inc/languages/english/admin/style_themes.lang.php

```diff
@@ -188,7 +188,7 @@
 $l['error_missing_stylesheet_extension'] = "This stylesheet must end with the correct file extension, for example, {1}<em>.css</em>";
 $l['error_invalid_parent_theme'] = "The selected parent theme does not exist. Please select a valid parent theme.";
 $l['error_invalid_templateset'] = "The selected template set does not exist. Please select a valid template set.";
-$l['error_invalid_color'] = "The defined color set \"{1}\" is either having invalid character(s) or not in prescribed format. Only alpha-numeric characters are allowed for naming separated by '='.";
+$l['error_invalid_color'] = "The defined color set \"{1}\" is either having invalid character(s) or not in prescribed format. Only numbers and unicode characters are allowed for naming separated by '='.";
```

Comment:
Minor nitpick: maybe "Unicode letters" is more accurate than "unicode characters" - a letter is more specifically something you'd use in an ordinary word, whereas a character could be anything.